### PR TITLE
Ability to send emails from the reminders framework

### PR DESCRIPTION
**Pull request by gcapalbo**
Simple Reminders UI:
email_simple

Advanced Reminders UI:
email_advanced

Broadcast UI (this needs to be updated to use crispy forms eventually, but that's a separate effort):
email_broadcast

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 16 errors:

* corehq/apps/reminders/forms.py, line 1708 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/forms.py, line 1721 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/views.py, line 220 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/views.py, line 220 - E502 the backslash is redundant between brackets
* corehq/apps/reminders/views.py, line 221 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/views.py, line 222 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/views.py, line 222 - E502 the backslash is redundant between brackets
* corehq/apps/reminders/views.py, line 223 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/views.py, line 279 - E502 the backslash is redundant between brackets
* corehq/apps/reminders/views.py, line 280 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/views.py, line 281 - E502 the backslash is redundant between brackets
* corehq/apps/reminders/views.py, line 282 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/views.py, line 283 - E501 line too long (117 > 115 characters)
* corehq/apps/reminders/static/reminders/ko/reminders.manage.js, line 369 - Missing semicolon.
* corehq/apps/reminders/static/reminders/ko/reminders.manage.js, line 410 - Bad line breaking before '||'.
* corehq/apps/reminders/static/reminders/ko/reminders.manage.js, line 411 - Bad line breaking before '||'.

**Comment by gcapalbo** [comment]
@nickpell code review buddy

File: corehq/apps/reminders/views.py [hunk 1]
```diff
@@ -213,7 +217,10 @@ def add_one_time_reminder(request, domain, handler_id=None):
                 day_num = 0,
                 fire_time = time(0,0),
                 form_unique_id = form.cleaned_data.get("form_unique_id") if content_type == METHOD_SMS_SURVEY else None,
-                message = {handler.default_lang : form.cleaned_data.get("message")} if content_type == METHOD_SMS else {},
+                message = ({handler.default_lang: form.cleaned_data.get("message")}
```
Comment (dimagimon) [inline corehq/apps/reminders/views.py]: E251 unexpected spaces around keyword / parameter equals

File: corehq/apps/reminders/views.py [hunk 2]
```diff
@@ -210,11 +214,15 @@ def add_one_time_reminder(request, domain, handler_id=None):
             handler.start_datetime = form.cleaned_data.get("datetime")
             handler.start_offset = 0
             handler.events = [CaseReminderEvent(
-                day_num = 0,
-                fire_time = time(0,0),
-                form_unique_id = form.cleaned_data.get("form_unique_id") if content_type == METHOD_SMS_SURVEY else None,
-                message = {handler.default_lang : form.cleaned_data.get("message")} if content_type == METHOD_SMS else {},
-                callback_timeout_intervals = [],
+                day_num=0,
+                fire_time=time(0,0),
```
Comment (dimagimon) [inline corehq/apps/reminders/views.py]: E231 missing whitespace after ','

**Pull request merged**